functional/MBCS_Tests/CLDR_11/build.xml

```diff
@@ -31,7 +31,7 @@ limitations under the License.
 		<mkdir dir="${build}" />
 	</target>
 
-	<target name="compile" depends="init" description="Using ${JAVA_VERSION} java compile the source  ">
+	<target name="compile" depends="init" description="Using ${JDK_VERSION} java compile the source  ">
```

Comment:
Just for the readability, can we change this to `Using java${JDK_VERSION} to compile the source `